pkgs/development/python-modules/geojson-pydantic/default.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core,
+  pytestCheckHook,
+  pydantic,
+  shapely,
+}:
+
+buildPythonPackage rec {
+  pname = "geojson-pydantic";
+  version = "2.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "developmentseed";
+    repo = "geojson-pydantic";
+    tag = version;
+    hash = "sha256-SOo4Hs1WMUerMLMgjzUVuDkUXCuiStJ4P9iMSpUF8Uw=";
+  };
+
+  nativeBuildInputs = [ flit-core ];
+
+  dependencies = [ pydantic ];
+
+  pythonImportsCheck = [ "geojson_pydantic" ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    shapely
+  ];
+
+  meta = {
+    changelog = "https://github.com/developmentseed/geojson-pydantic/blob/${version}/CHANGELOG.md";
+    description = "geojson_pydantic provides a suite of Pydantic models matching the GeoJSON specification rfc7946. Those models can be used for creating or validating geojson data";
```

```suggestion
    description = "Suite of Pydantic models matching the GeoJSON specification RFC 7946";
```

https://nixos.org/manual/nixpkgs/stable/#var-meta-description

If you want to add more information, feel free to use [`longDescription`](https://nixos.org/manual/nixpkgs/stable/#var-meta-longDescription).